docs/useful_tools.md

```diff
@@ -255,6 +255,36 @@ Examples:
   python tools/analyze_logs.py log.json --keys loss --legend loss
   ```
 
+### Model conversion
+
+We provide scripts to convert pretrain models provided by other repos to MMSegmentation format.
+
+#### vit/swin/mit models
```

Comment:
```suggestion
#### ViT Swin MiT Transformer Models
```

Reply:
Agree, the first letter should be uppercase letter.